EIPS/eip-crosschain-id.md

```diff
@@ -0,0 +1,131 @@
+
+## Backwards Compatibility
+
+Crosschainid can be backward compatible with EIP-155.  The crosschain id contains a 4 byte segment to record the chainid based on EIP-155.
+
+## Test Cases
+
+The test cases should cover the generation of crosschain id, the parsing of an crosschain id to ensure the various segments of the crosschain id meets the specification.
+For the crosschain registration smart contract, the test cases should cover the testing of all functions and also need to ensure that the security of each functions are properly implemented.
+
+## Implementation
+
+The implementation requires the following
+
+* Generate the crosschain id based on the format defined in the specification.  
+* Parse the crosschain id based on the format defined in the spefication.
+* For metadata asssociated with the crosschainid, the medadata needs to be registered with a smart contract.  The smart contract should implement the functions defined in the speficification.
+* Once a smart contract is deployed, the owner of the smart contract can publish the smart contract address, ABI and also crosschain id specification version.
+* Once a crosschain id is registered in the smart contract with extended metadata, there should be a waiting period for the community to review the registration and raise objection if there is inconsistency.
+
+## Security Considerations
+
+Security are considered in the following areas:
+Collision of crosschain id:  Two blockchains can contain the same crosschain id and hence making the mistakenly transfer assets to a wrong blockchain.  
+This security concern is addressed by comparing the hash of the crosschain id with the hash of the genesis block.  It it matches, then the crosschain id is verified.  If not, the crosschain id can be compared with the forked blockhash.  If none of the blockhash match the crosschain id hash, then the crosschain id cannot be verified.
+
+Preventing relay attack: Although crosschain id by itself is different from chainid and it is not signed into blockchain transaction, the crosschain id can still be used for presenting relay attack. An application that handles crosschain transaction can verified the crosschain id with its blockhash and decide whether the tranaction is valid or not. Any transaction with a non-verifiable crosschain id will be rejected.
+
+The crosschain-id are not required be signed into blockchaid tx.
+For blockchains that do not cryptographically sign crosschain id into the blocks, the crosschain id cannot be verified with the blocks itself and have to be verified with external smart contract address and offchain utilities implemented based on the crosschain id specification.
+
+## Copyright
+
+Copyright and related rights waived via [CC0](https://creativecommons.org/publicdomain/zero/1.0/).
+
+EEA copyright waived
```

Comment:
```suggestion
```
I don't know anything about EEA copyrights, but I believe every EIP currently has the exact same copyright section and I'm a bit loath to have one be a bit different (for any reason).  If you are interested in discussing adding `EEA copyright waived` more generally to EIPs, that is a welcome discussion and I recommend opening an issue or a PR against the template.

Reply:
This is from EEA legal team.  I will pass your comments to the legal team.